openedx/adg/admin_dashboard/performance/urls.py

```diff
@@ -0,0 +1,17 @@
+"""
+Urls for `performance` app
+"""
+from django.conf import settings
+from django.urls import path, re_path
+from django.views.generic import TemplateView
+
+urlpatterns = [
+    path(
+        '',
+        TemplateView.as_view(template_name='adg/admin_dashboard/performance/performance_tracking.html')
+    ),
+    re_path(
+        r'{}/grades/'.format(settings.COURSE_KEY_PATTERN),
```

Comment:
can we do it something like this?

`rf'{settings.COURSE_KEY_PATTERN}/grades/'`

Reply:
Never tried it, But yes we can. Updated!